### ISSUE TITLE
Suggestion for improvement of the chapter overview for blocked manga in the app

### ISSUE THREAD

**Issue by Forgrim** [proxer/ProxerAndroid#44]
#### This is what I request as a feature:  
Bring back the chapter overview for manga that are blocked in the app, **with a note or indicator that they are not available**.  

My reasons:  *As a User I want...*
+ *... to **track** the reading progress in the app.*
+ *... to see how many chapters are available.*
+ *... see if there are new chapters avalible (for example, on official sources or the website).*
+ *... to open and read the Chapter at **offical sources**. (like the redirections to the webtoon app)*

**Comment by rubengees** [comment]
Yes, that is totally reasonable. We had to remove the chapters quickly so the current solution was the easiest. We plan to follow up and improve UX exactly like you describe: Link official sources where possible and otherwise show an error with the ability to track.

**Comment by At-M** [comment]
Any update on this issue?

it's been >9 months by now, and atleast another checkbox within the bookmarks like "hide blocked content" should be fast enough to implement, since there should be some kind of marker in the database for each manga anyways
